### DISCUSSION TITLE
作りたい機能、やってみたいこと

### DISCUSSION THREAD

**Discussion by satsukizzz** [satsukizzz/n-playspace-http#7] [General]
教材のほかに作りこんでみたい機能があったら、こちらに書いてみましょう～

はっきりしてなくても構いません！そんなものでも、皆さんが会話しているうちにアイディアが具体的になるかもしれません！

kenton116さんより https://github.com/satsukizzz/n-playspace-http/discussions/5#discussioncomment-1441602
別スレッド化しました (_ _)

**Comment by kenton116** [comment]
slackのTODOボットをhttpアプリにしたいです！
